### DISCUSSION TITLE
Is there a recommended way to have a service locator or IOC container in serverpod ?

### DISCUSSION THREAD

**Discussion by rotoclabs** [serverpod/serverpod#2560] [Q&A] [answered]
I could potentially use `ioc_container` / `get_it` package, but is there something built into serverpod to do this or a roadmap where one could get included ?

**Comment by SandPod** [comment, marked as answer]
Hey @robythomas 👋 

There currently is nothing on the roadmap to add a service locator to Serverpod.
You are free you use any package that satisfies your requirements of the project.
